apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V27__Create_Enrollmenttable.sql

```diff
@@ -0,0 +1,141 @@
+	FROM "Beneficiaries"
+	WHERE "beneEnrollmentReferenceYear" is not null;
+
+INSERT INTO "Enrollments"
+    SELECT  "beneficiaryId", CONCAT("beneEnrollmentReferenceYear", '-08') as "yearMonth", "fipsStateCntyAugCode",
+    "medicareStatusAugCode", "entitlementBuyInAugInd", "hmoIndicatorAugInd",
+    "partCContractNumberAugId", "partCPbpNumberAugId", "partCPlanTypeAugCode",
+    "partDContractNumberAugId", "partDPbpNumberAugId", "partDSegmentNumberAugId",
+    "partDRetireeDrugSubsidyAugInd", "medicaidDualEligibilityAugCode", "partDLowIncomeCostShareGroupAugCode"
+	FROM "Beneficiaries"
+	WHERE "beneEnrollmentReferenceYear" is not null;
+
+INSERT INTO "Enrollments"
+    SELECT  "beneficiaryId", CONCAT("beneEnrollmentReferenceYear", '-09') as "yearMonth", "fipsStateCntySeptCode",
+    "medicareStatusSeptCode", "entitlementBuyInSeptInd", "hmoIndicatorSeptInd",
+    "partCContractNumberSeptId", "partCPbpNumberSeptId", "partCPlanTypeSeptCode",
+    "partDContractNumberSeptId", "partDPbpNumberSeptId", "partDSegmentNumberSeptId",
+    "partDRetireeDrugSubsidySeptInd", "medicaidDualEligibilitySeptCode", "partDLowIncomeCostShareGroupSeptCode"
+	FROM "Beneficiaries"
+	WHERE "beneEnrollmentReferenceYear" is not null;
+
+INSERT INTO "Enrollments"
+   SELECT  "beneficiaryId", CONCAT("beneEnrollmentReferenceYear", '-10') as "yearMonth", "fipsStateCntyOctCode",
+    "medicareStatusOctCode", "entitlementBuyInOctInd", "hmoIndicatorOctInd",
+    "partCContractNumberOctId", "partCPbpNumberOctId", "partCPlanTypeOctCode",
+    "partDContractNumberOctId", "partDPbpNumberOctId", "partDSegmentNumberOctId",
+    "partDRetireeDrugSubsidyOctInd", "medicaidDualEligibilityOctCode", "partDLowIncomeCostShareGroupOctCode"
+	FROM "Beneficiaries"
+	WHERE "beneEnrollmentReferenceYear" is not null;  
+
+INSERT INTO "Enrollments"
+    SELECT  "beneficiaryId", CONCAT("beneEnrollmentReferenceYear", '-11') as "yearMonth", "fipsStateCntyNovCode",
+    "medicareStatusNovCode", "entitlementBuyInNovInd", "hmoIndicatorNovInd",
+    "partCContractNumberNovId", "partCPbpNumberNovId", "partCPlanTypeNovCode",
+    "partDContractNumberNovId", "partDPbpNumberNovId", "partDSegmentNumberNovId",
+    "partDRetireeDrugSubsidyNovInd", "medicaidDualEligibilityNovCode", "partDLowIncomeCostShareGroupNovCode"
+	FROM "Beneficiaries"
+	WHERE "beneEnrollmentReferenceYear" is not null;  
+
+INSERT INTO "Enrollments"
+    SELECT  "beneficiaryId", CONCAT("beneEnrollmentReferenceYear", '-12') as "yearMonth", "fipsStateCntyDecCode",
+    "medicareStatusDecCode", "entitlementBuyInDecInd", "hmoIndicatorDecInd",
+    "partCContractNumberDecId", "partCPbpNumberDecId", "partCPlanTypeDecCode",
+    "partDContractNumberDecId", "partDPbpNumberDecId", "partDSegmentNumberDecId",
+    "partDRetireeDrugSubsidyDecInd", "medicaidDualEligibilityDecCode", "partDLowIncomeCostShareGroupDecCode"
+	FROM "Beneficiaries"
+	WHERE "beneEnrollmentReferenceYear" is not null;
```

Comment:
Are there any `Beneficiaries` records where `beneEnrollmentReferenceYear` is `NULL`? If so, what does that indicate about the bene record and how should we handle it here?

Reply:
what do u propose we should do if its null?

Reply:
Dunno! Depends on _why_ it's null.

Reply:
From our data providers: As of now, NULL means a beneficiary is accredited but not yet enrolled. We continue to get additional clarity on this.